packages/components/fields/async-select-field/src/async-select-field.spec.js

```diff
@@ -191,6 +191,27 @@ describe('when showing an info button', () => {
 });
 
 describe('when field is touched and has errors', () => {
+  it('should render an id for the error container that is based on the component id', async () => {
+    const { container, findByText } = renderAsyncSelectField({
+      touched: true,
+      errors: { custom: true },
+      renderError: () => 'Custom error',
+    });
+    await findByText(/custom error/i);
+    expect(
+      container.querySelector('#async-select-field-errors')
```

Comment:
Nit: wondering if we should `queryByRole` on the status role over the id of the div.

Reply:
the idea is that this is specifically testing that the desired id is present and formed correctly.  The next test does the more 'rtl approved' testing with roles, etc